c_binding/include/libstoragemgmt/libstoragemgmt_plug_interface.h

```diff
@@ -1249,6 +1250,19 @@ lsm_system LSM_DLL_EXPORT *lsm_system_record_alloc(const char *id,
                                                    const char *plugin_data);
 
 /**
+ * New in version 1.3. Set firmware version.
+ * @param[in] id            Id
+ * @param[in] sys           System to update.
+ * @param[in] fw_ver        Firmware version string.
+ *                          Will got LSM_ERR_INVALID_ARGUMENT for
```

Comment:
Change line 1257 comment to something like "Caller will get LSM_ERR_INVALID_ARGUMENT..."